pod2daemon/Makefile

```diff
@@ -85,7 +85,7 @@ ifeq ($(ARCH), $(filter $(ARCH),amd64))
 CGO_ENABLED=1
 TAGS=fipsstrict
 GOEXPERIMENT=boringcrypto
-REGISTRAR_BUILD_CMD=$(REGISTRAR_TIGERA_BUILD_CMD)
+REGISTRAR_BUILD_CMD=$(REGISTRAR_TIGERA_BUILD_CMD) $(LDFLAGS_CGO_STATIC)
```

Comment:
`pod2daemon/bin/node-driver-registrar-amd64` is still dynamically linked with this change for me. @coutinhop Can you double check if that binary is statically or dynamically linked in your workspace after your change?

Reply:
That's right, good catch @hjiawei! I thought I had seen it static, but on double checking I see it's not. Will fix, thanks